docs/source/blog/displacement-vectors.md

```diff
@@ -0,0 +1,47 @@
+---
+blogpost: true
+date: October 27, 2025
+author: Carlo Castoldi
+location: Milan, Italy
+category: update
+language: English
+---
+
+# Improving displacement vectors
+
+Computing locomotion features is a core utility since early versions of `movement`, when they where first introduced by [@lochhh](https://github.com/lochhh) in [#106](https://github.com/neuroinformatics-unit/movement/pull/106) for [v0.0.13](https://github.com/neuroinformatics-unit/movement/releases/tag/v0.0.13)—we are now at [v0.10.0](https://github.com/neuroinformatics-unit/movement/releases/tag/v0.10.0).
+For a library focused on tracking analysis and motion quantification, it's essential that key information, such as the total distance traveled or the tracking point's location in adjacent frames (next or previous), can be easily calculated. That's why we introduced {func}`compute_displacement <movement.kinematics.compute_displacement>` so early on: it's a function that some of you may have used to have a grasp of your tracking data, and it is also featured in our {ref}`compute and visualise kinematics <sphx_glr_examples_compute_kinematics.py>` example<!--#compute-displacement-vectors heading-->.
+
+Its implementation, however, produces results of complex interpretability. It defines, for a given individual and keypoint at timestep `t`, a vector that points to the opposite direction of its previous position at time `t-1`.
```

Comment:
I would expand this paragraph to provide some more motivation for the change, sth like this:

```suggestion
Its original implementation, however, produced results that were difficult to interpret. For a given individual and keypoint at timestep `t`, displacement was defined as the vector pointing from the previous position at `t-1` to the current position at `t`. This definition is somewhat counter-intuitive: it indicates where the point _came from_ rather than where it is _going_, and it does not allow for computing displacement vectors that point forward in time, i.e. from `t` to `t+1`.
```

Reply:
The previous situation was a bit more counter-intuitive than computing vectors that point where it  came from (i.e. backward displacements). The direction was opposite too.

```suggestion
Its original implementation, however, produced results that were difficult to interpret. For a given individual and keypoint at timestep `t`, displacement was defined as the vector pointing from the previous position at `t-1` to the current position at `t`. This definition is somewhat counter-intuitive: it indicates the _opposite_ direction from where the point _came from_, and it does not allow for computing displacement vectors that point forward in time, i.e. from `t` to `t+1`.
```